pyaedt/circuit.py

```diff
@@ -372,7 +372,7 @@ def create_schematic_from_netlist(self, file_to_import):
         return True
 
     @pyaedt_function_handler()
-    def read_ibis(self, path):
+    def get_ibis_model_from_file(self, path):
```

Comment:
@maxcapodi78 I don't think we need any deprecation warning here as the method was not working properly.